src/i2c.rs

```diff
@@ -116,6 +119,8 @@ impl<PINS> I2c<I2C1, PINS> {
 }
 
 impl<PINS> BlockingI2c<I2C1, PINS> {
+    /// Creates a blocking I2C1 object on pins PB6 and PB7 or PB8 and PB9 (if remapped)
+    /// embedeed-hal compatible!
```

Comment:
There's a typo and this looks a bit odd when rendered:

![image](https://user-images.githubusercontent.com/800791/76229908-d4f94880-621a-11ea-9b36-893482d32977.png)

That said, it's a good idea to mention embedded hal compatibility. Maybe:

```suggestion
    /// Creates a blocking I2C1 object on pins PB6 and PB7 or PB8 and PB9 (if remapped) using the embedded-hal `BlockingI2c` trait.
```